src/libostree/ostree-sysroot-deploy.c

```diff
@@ -2866,6 +2929,17 @@ sysroot_finalize_deployment (OstreeSysroot     *self,
         return FALSE;
     }
 
+  static const gchar * const SEMODULE_CMD[] = {
+    "semodule", "-N", "--rebuild-if-modules-changed"
+  };
+  gint exit_status;
+  if (!run_in_deployment(deployment_dfd, SEMODULE_CMD,
+                         sizeof(SEMODULE_CMD) / sizeof(*SEMODULE_CMD),
+                         &exit_status, error))
+    return FALSE;
+  if (!g_spawn_check_exit_status(exit_status, NULL))
+    g_message ("Failed to refresh SELinux policy - the policy contents may be inconsistent");
+
```

Comment:
I think we need to gate all this on `HAVE_SELINUX`.

Reply:
I guess you're right.. While technically the code doesn't depend on libselinux, if ostree isn't built with `HAVE_LIBSELINUX`, then it can't relabel files in the deployment and thus attempting any selinux-policy management would be pointless anyway...

Reply:
Wait, sorry, actually it's `HAVE_SELINUX`. not `HAVE_LIBSELINUX`, and it means simply "SELinux support", not "use libselinux", so it's even more obvious that this should be under `HAVE_SELINUX` :)